infra/recipes/terraform/azure/modules/helm_release/main.tf

```diff
@@ -0,0 +1,54 @@
+locals {
```

Comment:
/**
Module for defining helm release for openhouse services. Installs tables and housetables service using the corresponding helm charts. Tables and housetables service images are referred from Azure container registry (ACR).
*/